app/commands/exercise/update_tags.rb

```diff
@@ -12,6 +12,8 @@ def tags
 
     new_tags = (solution_tags - exercise_tags).map do |tag|
       Exercise::Tag.find_or_create_by!(tag:, exercise:)
+    rescue ActiveRecord::RecordNotUnique
+      Exercise::Tag.find_by!(tag:, exercise:)
```

Comment:
```suggestion
      Exercise::Tag.find_create_or_find_by!(tag:, exercise:)
```

Reply:
Did you mean to suggest this for the original line, not for the exception?

Reply:
I think this replaces all three lines.

Reply:
The `or_find_by!` does exactly this under the hood.

Reply:
Okay good!